apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -124,7 +124,11 @@ export class SendMessageEmail extends SendMessageBase {
       return;
     }
 
-    const overrides: Record<string, any> = command.overrides[integration?.providerId] || {};
+    const overrides: Record<string, any> = Object.assign(
+      {},
+      command.overrides,
+      command.overrides[integration?.providerId]
```

Comment:
```suggestion
      command.overrides['email'] || {},
      command.overrides[integration?.providerId] || {}
```

Reply:
if we will add '|| {}' then only the last one will merge and it will result in: 
command.overrides[integration?.providerId] OR {}
The previous one merged all the entities when the last one is the most will override the first one.